docs/user-guide/cli-install-configure-zosmf.md

```diff
@@ -0,0 +1,33 @@
+
+## Installing and configuring z/OSMF
+
+Zowe CLI was designed and tested to integrate with z/OSMF running on IBM version 2.5 z/OS mainframe systems. To use Zowe CLI, ensure that your z/OS system meets the requirements that are described in the following table:
+
+| Requirement        | Description |
+| ----------- | ----------- |
+| AXR (System Rexx) | The AXR (System Rexx) component lets z/OS perform Incident Log tasks. It also lets REXX execs execute outside of conventional TSO and batch environments. <br/>For more information, see [Communicating with System REXX](https://www.ibm.com/docs/en/zos/2.5.0?topic=command-communicating-system-rexx) on the IBM Documentation. |
+| CEA (Communications Enabled Applications)  Server | CEA server is a co-requisite for the CIM server. The CEA server lets z/OSMF deliver z/OS events to C-language clients. <br/>z/OSMF requires the CEA server to perform the following types of tasks: <ul><li>Problem determination</li><li>Sysplex</li><li>z/OS classic interfaces</li> <li>z/OS Operator Console</li></ul> **Notes:** <ul><li>Start the CEA server before you start z/OSMF (the IZU* started tasks).</li><li> Set up CEA server in Full Function Mode and assign the TRUSTED attribute to the CEA started task.</li></ul> For more information, see [Customizing for CEA](https://www.ibm.com/docs/en/zos/2.5.0?topic=test-customizing-cea) on the IBM Documentation. |
+| CIM (Common Information Model) Server | z/OSMF requires the CIM server to perform the following types of tasks: <ul><li>Capacity provisioning </li><li> Problem determination </li><li> Workload management </li></ul> **Note:** Start the CIM server before you start z/OSMF (the IZU* started tasks). <br/> For more information, see [Configuring the CIM server for your system](https://www.ibm.com/docs/en/zos/2.5.0?topic=configurations-configuring-cim-server-your-system)  on the IBM Documentation. <br/> For more information on how to perform asynchronous operations, review the [required authorizations for z/OSMF](https://www.ibm.com/docs/en/zos/2.5.0?topic=services-zos-jobs-rest-interface#izuhpinfo_api_restjobs__RequiredAuthorizationsForRestServices__title__1) on the IBM Documntation.|
+| Console Command | The `CONSOLE` and `CONSPROF` commands must exist in the authorized command table. |
+| Java version | IBM 64-bit SDK for z/OS, Java Technology Edition V7.1 or higher is required. However, we experienced problems accessing z/OSMF 2.2 using Java version 8. If you use z/OSMF 2.3, Java version 8.0_64 is required. <br/>```SYS1.PARMLIB member  IZUPRMxx.... JAVA_HOME('/sys/java64bt/v7r1m0/usr/lpp/java/J7.1_64'/etc/profile ............................................... export JAVA_HOME=/sys/java31bt/v7r1m0/usr/lpp/java/J7.1 ```<br/>For more information, see [Software prerequisites for z/OSMF](https://www.ibm.com/docs/en/zos/2.5.0?topic=zosmf-software-prerequisites) on the IBM Documentation. |
```

Comment:
@zFernand0 

I don't know much about configuring z/OSMF, but it might be useful to explain what the code block is for. I assume it's to make sure that a system uses Java 7.1 if it's running z/OSMF 2.2. Also, z/OS 2.2 and 2.3 are out of support, so maybe we could just remove this instruction - or should it be updated? 

I'm not sure that the code block is correct either:
- There is an unterminated bracket after "JAVA_HOME('..."
- Is /etc/profile supposed to be on the same line as "JAVA_HOME('..."?
- Why are there 47 periods before the export command?

Reply:
> There is an unterminated bracket after "JAVA_HOME('..."

yeah, this needs to be either fixed or removed

> Is /etc/profile supposed to be on the same line as "JAVA_HOME('..."?

Not really, 😋  See below for a fixed version (if we want to use it)

> Why are there 47 periods before the export command?

I believe the intention was to align the contents of the table withouth adding a new line `¯\_(ツ)_/¯`

---

I agree that we can probably update this to mention the minimum version of Java.
And then quickly link to the IBM z/OS 2.5 documentation
Here is what I think we can simplify this to

---

IBM® 64-bit SDK for z/OS®, Java™ Technology Edition, V8 SR4 FP10 (5655-DGH).

For more information, see [Software prerequisites for z/OSMF](https://www.ibm.com/docs/en/zos/2.5.0?topic=zosmf-software-prerequisites) on the IBM Documentation.

---

I know we are going to remove that block of code, however, I believe this is what it meant to indicate:

```
/* ================================================================= */
/*                Contents of SYS1.PARMLIB(IZUPRMxx)                 */
/* ================================================================= */
JAVA_HOME('/sys/java64bt/v8r4m10/usr/lpp/java/J8.4_10')

    PLUGINS( ISPF, RESOURCE_MON, SOFTWARE_MGMT, SYSPLEX_MGMT)

/* ================================================================= */
/*  For manual configuration you have to add the                     */
/*  following line to your `/etc/profile`                            */
/*                                                                   */
/*   export JAVA_HOME='/sys/java64bt/v8r4m10/usr/lpp/java/J8.4_10'   */
/* ================================================================= */

```

Reply:
@adam-wolfe @zFernand0 - would this be better/accurate for the table?

![image](https://github.com/zowe/docs-site/assets/109533923/adb173ab-3cab-434f-a324-cc0a58ecdb8e)